specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/HybridCompute.json

```diff
@@ -2186,86 +2164,254 @@
-          "description": "Indicates which kind of VM fabric the instance is an instance of, such as HCI or SCVMM etc.",
-          "enum": [
-            "AVS",
-            "HCI",
-            "SCVMM",
-            "VMware"
-          ],
-          "x-ms-enum": {
-            "name": "privateCloudKind",
-            "modelAsString": true
-          }
+          "description": "The list of extensions affiliated to the machine"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/Identity"
+        },
+        "kind": {
+          "$ref": "#/definitions/ArcKindEnum "
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Describes a hybrid machine."
+    },
+    "ArcKindEnum ": {
+      "type": "string",
+      "description": "Indicates which kind of Arc machine placement on-premises, such as HCI, SCVMM or VMware etc.",
+      "enum": [
+        "AVS",
+        "HCI",
+        "SCVMM",
+        "VMware",
+        "EPS",
+        "GCP",
+        "AWS"
+      ],
+      "x-ms-enum": {
+        "name": "ArcKindEnum ",
+        "modelAsString": true
+      }
+    },
+    "MachineUpdate": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/Identity"
+        },
+        "kind": {
+          "$ref": "#/definitions/ArcKindEnum "
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/MachineUpdateProperties",
+          "description": "Hybrid Compute Machine properties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceUpdate"
+        }
+      ],
+      "description": "Describes a hybrid machine Update."
+    },
+    "MachineListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Machine"
+          },
+          "description": "The list of hybrid machines."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of Machines. Call ListNext() with this URI to fetch the next page of hybrid machines."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List hybrid machine operation response."
+    },
+    "LicenseProfile": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "properties": {
+            "esuProfile": {
+              "x-ms-client-flatten": true,
+              "$ref": "#/definitions/EsuProfileProperties",
+              "description": "Hybrid Compute ESU Profile properties"
+            },
+            "provisioningState": {
```

Comment:
i didnt see this in the examples. do we need it?

Reply:
ARM restriction